src/ParallelEnv.f90

```diff
@@ -152,46 +152,62 @@ SUBROUTINE ParEnvType_clear_absintfc(myPE)
+      PROCEDURE,PASS,PRIVATE :: bcast_SSK1_MPI_Env_type
       !> @copybrief ParallelEnv::bcast_SSK2_MPI_Env_type
       !> @copydetails ParallelEnv::bcast_SSK2_MPI_Env_type
       PROCEDURE,PASS,PRIVATE :: bcast_SSK2_MPI_Env_type
+      !> @copybrief ParallelEnv::bcast_SSK3_MPI_Env_type
+      !> @copydetails ParallelEnv::bcast_SSK3_MPI_Env_type
+      PROCEDURE,PASS,PRIVATE :: bcast_SSK3_MPI_Env_type
+      !> @copybrief ParallelEnv::bcast_SSK4_MPI_Env_type
+      !> @copydetails ParallelEnv::bcast_SSK4_MPI_Env_type
+      PROCEDURE,PASS,PRIVATE :: bcast_SSK4_MPI_Env_type
       !> @copybrief ParallelEnv::bcast_SDK1_MPI_Env_type
       !> @copydetails ParallelEnv::bcast_SDK1_MPI_Env_type
       PROCEDURE,PASS,PRIVATE :: bcast_SDK1_MPI_Env_type
       !> @copybrief ParallelEnv::bcast_SDK2_MPI_Env_type
       !> @copydetails ParallelEnv::bcast_SDK2_MPI_Env_type
       PROCEDURE,PASS,PRIVATE :: bcast_SDK2_MPI_Env_type
-      !> @copybrief ParallelEnv::bcast_SSK3_MPI_Env_type
-      !> @copydetails ParallelEnv::bcast_SSK3_MPI_Env_type
-      PROCEDURE,PASS,PRIVATE :: bcast_SSK3_MPI_Env_type
       !> @copybrief ParallelEnv::bcast_SDK3_MPI_Env_type
       !> @copydetails ParallelEnv::bcast_SDK3_MPI_Env_type
       PROCEDURE,PASS,PRIVATE :: bcast_SDK3_MPI_Env_type
-      !> @copybrief ParallelEnv::bcast_SSK4_MPI_Env_type
-      !> @copydetails ParallelEnv::bcast_SSK4_MPI_Env_type
-      PROCEDURE,PASS,PRIVATE :: bcast_SSK4_MPI_Env_type
       !> @copybrief ParallelEnv::bcast_SDK4_MPI_Env_type
       !> @copydetails ParallelEnv::bcast_SDK4_MPI_Env_type
       PROCEDURE,PASS,PRIVATE :: bcast_SDK4_MPI_Env_type
       !>
-      GENERIC :: bcast => bcast_SLK0_MPI_Env_type, &
+      GENERIC :: bcast => bcast_SNK0_MPI_Env_type, &
+                          bcast_SNK1_MPI_Env_type, &
+                          bcast_SLK0_MPI_Env_type, &
+                          bcast_SLK1_MPI_Env_type, &
+                          bcast_SSK1_MPI_Env_type, &
                           bcast_SSK2_MPI_Env_type, &
+                          bcast_SSK3_MPI_Env_type, &
+                          bcast_SSK4_MPI_Env_type, &
                           bcast_SDK1_MPI_Env_type, &
                           bcast_SDK2_MPI_Env_type, &
-                          bcast_SSK3_MPI_Env_type, &
                           bcast_SDK3_MPI_Env_type, &
-                          bcast_SSK4_MPI_Env_type, &
                           bcast_SDK4_MPI_Env_type
       !> @copybrief ParallelEnv::allReduceR_MPI_Env_type
       !> @copydetails  ParallelEnv::allReduceR_MPI_Env_type
       PROCEDURE,PASS :: allReduce => allReduceR_MPI_Env_type
       !> @copybrief ParallelEnv::allReduceI_MPI_Env_type
       !> @copydetails  ParallelEnv::allReduceI_MPI_Env_type
       PROCEDURE,PASS :: allReduceI => allReduceI_MPI_Env_type
-      !GENERIC :: allReduce => allReduceR_MPI_Env_type,allReduceI_MPI_Env_type
+      !GENERIC :: allReduce => allReduceI_MPI_Env_type,allReduceI_MPI_Env_type
```

Comment:
What's going on here?  You got rid of ```allReduceR``` and added two ```AllReduceI```'s?

Reply:
Oh... didn't mean to do this, but it's commented out in both versions, so it's not actually affecting anything.